package.json

```diff
@@ -20,6 +20,7 @@
     "axios": "1.6.2",
     "commander": "^6.0.0",
     "form-data": "4.0.0",
+    "openid-client": "^5.6.1",
```

Comment:
I believe there was a dependency on the datadog-ci because of uploading source-maps. Since you deleted that, can you try to remove the dependency as well?

Reply:
I was not thinking of tracing.ts, but that probably can be removed as well

Reply:
Can we please revert changes to `tracing.ts`? Those environment variables are used to propagate the Datadog trace context when using content-cli.

Reply:
sure @promeris , will revert those

Reply:
so, I guess the dependency on datadog-ci is still necessary?

Reply:
For the tracing requirement it's not necessary, because that works directly with HTTP headers. Thanks for reverting